virttest/env_process.py

```diff
@@ -1227,6 +1227,17 @@ def preprocess(test, params, env):
     LOG.debug("KVM userspace version(qemu): %s", kvm_userspace_version)
     version_info["qemu_version"] = str(kvm_userspace_version)
 
+    #Get the SLOF version for power pc
```

Comment:
For what I want is to let it be configurable, I have 2 ideas:
1. Get the firmware version for each platform:
    - ovmf, aarch64: check edk2 version
    - PC machine: check seabios
    - PPC: check SLOF
    - s390x: I don't know which one
2. Make all collection versions can be customizable, qemu/kvm/libvirt/swtpm/firmware...

Cc @luckyh @YongxueHong

Reply:
Hi all, 
I listed the build info for all arches and feel free to let me know if there's any missing.
1.
aarch64:  edk2
x86_64: both seabios and edk2
s390x: need to confirm with bfu
2.
swtpm for all arches.